docs/application/dotnet/guides/nui/tableview.md

```diff
@@ -1,40 +1,45 @@
 # TableView
 
-The `TableView` class is a layout container for aligning child actors in a grid like layout.
+The [TableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TableView.html) class is a layout container for aligning child actors in a grid like layout.
 `TableView` constraints the x and y position, width, and height of the child actors.
 The Z position and depth are left intact, so that the three-dimensional model actors can also be laid out in a grid without losing their depth scaling.
 
-In this tutorial, the following subjects are covered:
+## Create a TableView
 
-[Creating a TableView](#1)<br>
-[TableView Properties](#2)<br>
+The following example illustrates how to create a `TableView` object:
 
-<a name="1"></a>
-## Creating a TableView
+**Figure: TableView example**
 
-The following example illustrates how to create a `TableView` object:
+![AlignSelf](media/tableview.png)
+
+1.  Add the following namespaces:
+
+    ```csharp
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```
+
+2.  Create `TableView` as the whole view by setting the resize policy to fill its parent. Add `TextLabel`s as cells of `TableView`:
```

Comment:
Changing to active voice and as per style guide recommendation:

Set the resize policy to fill its parent to create `TableView` as the whole view.